ui/k-info-panel/list_connections.kytos

```diff
@@ -0,0 +1,97 @@
+<template>
+    <k-table title="list of EVCs"
+        :headers="headers"
+        :rows="rows">
+    </k-table>
+
+</template>
+<script>
+module.exports = {
+  data(){
+    return {
+        headers: ['Name ', 'Endpoint A' , 'Tag A' , 'Endpoint Z' , 'Tag z' , 'Enabled' , 'Active'],
+
+        rows: this.list_EVCs()
+    }
+  },
+   mounted () {
+     console.log('inside the list connections: ' )
+   },
+  methods: {
+
+    list_EVCs()
+  {
+    var tableDataArr = [];
+
+     $.ajax(
+     {
```

Comment:
Indentation is inconsistent. Current standard for these files seems to be 2 spaces, not four.